packages/riipen-ui/src/components/AppBar.test.jsx

```diff
@@ -0,0 +1,92 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import AppBar from "./AppBar";
+
+describe("<AppBar>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<AppBar />);
+    } catch (theError) {
+      error = theError;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("sets correct default props", () => {
+    const defaultProps = new AppBar().type.defaultProps;
+
+    const wrapper = mount(<AppBar />);
+
+    const component = wrapper.find("AppBar").childAt(0);
+    expect(component.hasClass(defaultProps.color)).toEqual(true);
+    expect(component.hasClass(defaultProps.position)).toEqual(true);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("displays custom label from child", () => {
+      const label = "A label given as a child";
+      const child = <div>{label}</div>;
```

Comment:
I think you can actually just combine these. 

```js
const child = <div>A child</div>;

const wrapper = mount(<AppBar>{child}</AppBar>);

expect(wrapper.containsMatchingElement(child)).toBeTruthy();
```

since we aren't testing this specific label thing.